homework.md

```diff
@@ -44,37 +44,37 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used on the internet in websites for scripting, security and providing functionality.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Version Control, Sharing Code, Backing up projects.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You make your own copy of another repo on your own Github profile.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You download the repo to your pc.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+When you send a request to the owner of the repo to make changes
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It lets you see all changes made to the project.
```

Comment:
shows a list of all the commits made to a repository

Reply:
Hi I think I was answering in laymans terms - as in a repository is a project and commits is the changes made, But I really need to get better at using the technical terms so thank you for pointing that out.